build.gradle.kts

```diff
@@ -0,0 +1,36 @@
+import com.android.build.gradle.BaseExtension
+import com.android.build.gradle.api.AndroidBasePlugin
+import com.diffplug.gradle.spotless.SpotlessExtension
+
+plugins {
+    id("com.android.application") version "8.4.1" apply false
+    id("com.android.library") version "8.4.1" apply false
+    id("org.jetbrains.kotlin.android") version "2.0.0" apply false
+    id("com.vanniktech.maven.publish") version "0.28.0" apply false
+    id("com.diffplug.spotless") version "6.25.0" apply false
+}
+
+subprojects {
+    plugins.withType<AndroidBasePlugin>().configureEach {
+        extensions.configure<BaseExtension> {
+            compileSdk = 34
```

Comment:
It's `compileSdkVersion(34)`!

Please pay more attention to your changes if you want to be involved in OSS projects. At least make sure the changes are passed on your local, instead of amending these stupid changes once and once again.